Exercism/Gigasecond/Gigasecond.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercism.Gigasecond
+{
+    public static class Gigasecond
+    {
+        public static DateTime Date(DateTime birthDate)
+        {
+            return birthDate.AddSeconds(1000000000).ToUniversalTime();
```

Comment:
You shouldn't need `ToUniversalTime()`. In fact I think it will break if someone passes in a value that is *not* in UTC.